packages/@ember/-internals/glimmer/lib/helpers/gt.ts

```diff
@@ -0,0 +1,38 @@
+import { Arguments, CapturedArguments, VM } from '@glimmer/runtime';
+import { InternalHelperReference } from '../utils/references';
+
+/**
+@module ember
+*/
+
+/**
+  Compares the two given values with >
+
+  Example:
+
+  ```handlebars
+  {{gt age 17}}
+
+  {{! be true if `age` is > 17}}
+  ```
+
+  @public
+  @method gt
+  @for Ember.Templates.helpers
+  @since 2.7.0
+*/
+function gt({ positional: { references } }: CapturedArguments) {
+  let left = references[0].value();
+  if (left === undefined || left === null) {
+    return false;
+  }
+  let right = references[1].value();
+  if (right === undefined || right === null) {
+    return false;
```

Comment:
I think if right is `null`, gt can return `true` if one follows JS semantics:

![20181104-161146](https://user-images.githubusercontent.com/1205444/47966143-48031580-e04f-11e8-8bdf-4b06b9210f9a.png)

Reply:
TIL. Good catch.

Reply:
Updated. `null` when used in numeric contexts is treated as 0.
If I just compare `left > right` when right can be null, typescript complains, so I explicitly handle that case. However if someone knows how to make typescript accept that, I think it would be dryer.
I added tests cases.